### ISSUE TITLE
Detecting quokka to enable mocking, avoid remote calls

### ISSUE THREAD

**Issue by benfletcher** [wallabyjs/quokka#306]
### Issue description or question

What's the best way to detect that my node project is running under/via Quokka vs. natively/manually? I'm in a CLI node project that will use some default values instead and mock data instead of remote server calls. I see a few ways to detect (one example below), but don't want to rely on undocumented things that could change in a future version.

### Sample code

```javascript
if (process.env.quokka) {
  // quokka instance... 
  // mock remote server calls...
  // use default CLI arguments instead of process.env.argv
} else {
  // no quokka...
}
```

tl;dr Will Quokka continue to store settings in that location?

### Sample repository link

n/a

### Quokka.js Console Output

n/a

### Code editor version

n/a

### OS name and version

OSX Mojave

**Comment by ArtemGovorov** [comment]
```js
if (process.env.quokka) {
```

is the safe/best way to do it.
